vhdbuilder/packer/install-dependencies.sh

```diff
@@ -241,20 +209,8 @@ fi
 ls -ltr /opt/gpu/* >> ${VHD_LOGS_FILEPATH}
 
 installBpftrace
-echo "  - bpftrace" >> ${VHD_LOGS_FILEPATH}
-
-cat << EOF >> ${VHD_LOGS_FILEPATH}
-  - nvidia-docker2=${NVIDIA_DOCKER_VERSION}
-  - nvidia-container-runtime=${NVIDIA_CONTAINER_RUNTIME_VERSION}
-  - nvidia-gpu-driver-version=${GPU_DV}
-  - nvidia-fabricmanager=${GPU_DV}
```

Comment:
this one comes from L210 a bit up, could we change it to e.g. "nvidia-driver=${NVIDIA_DRIVER_IMAGE_TAG}"

this is a container we build ourselves but the tag contains the driver version

we can delete all these 4 like you've done, but would be great to replace

Reply:
@alexeldeib PTAL :)